### PR TITLE
Stub API queries

### PR DESCRIPTION
**Pull request by errfrom**
- wip: add high-level auction workflow pseudocode for L1
- `announceAuction`: return `AuctionInfo` of the announced auction
- stub `enterAuction`
- stub `discoverBidders`
- stub `authorizeBidders`
- stub `discoverSellerSignature`
- stub `queryStandingBidState`
- add stubbed queries to demo

### CODE REVIEW

File: src/Contract/AuthorizeBidders.purs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      }
+
+authorizeBiddersContract
+  :: AuthorizeBiddersContractParams
+  -> Contract (ContractOutput TransactionHash)
+authorizeBiddersContract = const authorizeBiddersContractStub
+
+authorizeBiddersContractStub :: Contract (ContractOutput TransactionHash)
+authorizeBiddersContractStub = do
+  txHash <- liftEffect $ randomSampleOne arbitrary
```
Comment (GeorgeFlerovsky) [inline src/Contract/AuthorizeBidders.purs]: Hmm. Not sure if randomizing the stub outputs is helpful to frontend team as they're building out their components.

We should ask them if they would prefer to have deterministic hard-coded stub outputs.

File: src/Contract/Types/VerificationKey.purs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+instance Show VerificationKey where
+  show (VerificationKey vkey) = "(VerificationKey " <> byteArrayToHex vkey <> ")"
+
+instance Arbitrary VerificationKey where
+  arbitrary =
+    vectorOf vkeyLength (chooseInt 0 255)
+      <#> VerificationKey <<< byteArrayFromIntArrayUnsafe
+
+vkeyLength :: Int
+vkeyLength = 32
```
Comment (GeorgeFlerovsky) [inline src/Contract/Types/VerificationKey.purs:42]: Magic numbers should have explanatory code comments, or unbroken links to docs explaining them.

**Review by GeorgeFlerovsky — APPROVED** [review]
Looks good to me :+1:

File: src/Contract/Types/VerificationKey.purs (hunk 2 above)
Comment (errfrom) [reply src/Contract/Types/VerificationKey.purs:42]: Addressed in https://github.com/mlabs-haskell/hydra-auction-offchain/pull/16/commits/6b34b0ff8b9c803506105f0ba9b55ec5e5bf201d

**Pull request merged**